### PR TITLE
✨Add prefix for generated users (continuation)

### PR DESCRIPTION
**Pull request by guillaumearnx**
Good evening,

as mentioned in the PR #1378, it would be interesting to have the possibility of generating a prefix for the users created on the different managers.

I see that in some managers, the method is overridden.
I therefore still wanted to include this option.
Ignored option for CPanel due to restrictions (max 16 characters). This is specified on the front and translated into a few languages.

Thank you so much

### CODE REVIEW

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
Please remove the changes as there is allready a PR for it

**Comment by guillaumearnx** [comment]
Sorry thats a bug unwanted frome me. Have a good day

File: src/modules/Servicehosting/html_admin/mod_servicehosting_server.html.twig [hunk 1]
```diff
@@ -112,7 +112,7 @@
             <div class="mb-3 row">
                 <label class="form-label col-3 col-form-label">{{ 'Users prefix'|trans }}:</label>
                 <div class="col">
-                    <input class="form-control" type="text" name="userprefix" value="{{ server.config.userprefix }}" placeholder="{{ 'Prefix for created users (especially for database)'|trans }}">
+                    <input class="form-control" type="text" name="userprefix" value="{{ server.config.userprefix }}" placeholder="{{ 'Prefix for created users (especially for database) | Ignored for CPanel'|trans }}">
```
Comment (BelleNottelling) [inline src/modules/Servicehosting/html_admin/mod_servicehosting_server.html.twig:115]: I'd probably add a `span` under this input with the `text-muted` class and write out something similar to "Some server managers may ignore this option due to limitations / requirements for usernames imposed by the control panel".

That way we don't need to remember to change this if any others end up not using the prefix & we are less likely to have people submitting a bug report if their 3rd party server manager doesn't use it

**Review by BelleNottelling — COMMENTED** [review]
The functional changes look good, I'd just prefer different wording for the configure page

**Comment by guillaumearnx** [comment]
indeed, this is preferable. thanks for the advice

File: src/modules/Servicehosting/html_admin/mod_servicehosting_server.html.twig [hunk 2]
```diff
@@ -113,6 +113,7 @@
                 <label class="form-label col-3 col-form-label">{{ 'Users prefix'|trans }}:</label>
                 <div class="col">
                     <input class="form-control" type="text" name="userprefix" value="{{ server.config.userprefix }}" placeholder="{{ 'Prefix for created users (especially for database)'|trans }}">
+                    <span class="text-muted">{{ 'Some server managers may ignore this option due to limitations / requirements for username imposed by the control panel.'|trans }}</span>
```
Comment (BelleNottelling) [inline src/modules/Servicehosting/html_admin/mod_servicehosting_server.html.twig:116]: Typo, I think "username" was supposed to be "usernames"

**Comment by guillaumearnx** [comment]
very likely not being an expert in English

**Review by BelleNottelling — APPROVED** [review]
Cheers, thanks for the PRs!

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**